teamcity.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-export NODE_OPTIONS="--max-old-space-size=8192"
+export NODE_OPTIONS="--max-old-space-size=16384"
```

Comment:
Ran into a memory error in teamcity. I think it's a little wild that we need this much memory on SDC. I'm planning to do some more 10% time on coming up with a plan to restructure SDC and potentially alter how we build these modules (e.g using esbuild instead of rollup).

Reply:
Hopefully this will help speed up build times and reduce memory useage

Reply:
wow, that is really bad

Reply:
☹️